README.md

```diff
@@ -533,9 +534,14 @@ For more, see [this SO thread](http://stackoverflow.com/questions/11058312/how-c
 <a name="interactive-rebase"></a>
 ### I need to combine commits
 
-You need to do something called an interactive rebase.
+Let's suppose you are working in a branch that is/will become a pull-request against `master`. In the simplest case when all you want to do is to combine *all* commits into a single one and you don't care about commit timestamps, you can reset and recommit. Make sure the master branch is up to date and all your changes committed, then:
 
-If you are working in a branch that is/will become a pull-request against `master`, you can rebase against your `master` branch. Make sure the master branch is up to date, then:
+```sh
+(my-branch)$ git reset --soft master
```

Comment:
This line assumes that you are on `my-branch` and want to squash it on top of `master`.

`git reset --soft` resets HEAD to `master` and leaves all changes between `master` and `my-branch` in the index, ready for commit. It would obviously have no effect if the trees at `master` and `my-branch` were identical, but then it's expected isn't it? (You would have to `--skip` all your commits in the rebase approach anyway.)